File: src/main/java/com/openvalue/boardgameratings/service/rating/RateRepository.java
package com.openvalue.boardgameratings.service.rating;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface RateRepository extends CrudRepository<RateEntity, Long> {

    List<RateEntity> findByBoardGameName(String boardGameName);
}

File: src/main/java/com/openvalue/boardgameratings/service/rating/RateRepository.kt
package com.openvalue.boardgameratings.service.rating

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface RateRepository : CrudRepository<RateEntity, Long> {
    fun findByBoardGameName(boardGameName: String): List<RateEntity>

}